recipes-wigwag/mbed-fcce/mbed-fcce/mbed-fcce/0001-Added-trusted-storage-to-Yocto-target.patch

```diff
@@ -0,0 +1,105 @@
+From 3904a9ccb468999c17883857af1869329713b155 Mon Sep 17 00:00:00 2001
+From: Yash Goyal <ygoyal@wigwag.com>
```

Comment:
wigwag email - should be using the @pelion.com email instead @ygoyal18 .
'

Reply:
Fixed.